app/controllers/concerns/api.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
This is only valid on ruby versions higher or equal than 2.3. Thus, I'd recommend adding the following on the rubocop file and removing all the comments:

```yaml
Style/FrozenStringLiteralComment:
  Enabled: false
```